app/hooks/useModalVisible.js

```diff
@@ -0,0 +1,12 @@
+import { useSelector } from "react-redux";
+import * as sel from "selectors";
+
+const useModalVisible = () => {
```

Comment:
This custom hook is not used anywhere.

Reply:
It's used in `BlurableContainer.jsx`:

https://github.com/decred/decrediton/pull/3278/files#diff-24b84bdbeaf013c28b29eec0c133f6dae393ad8231a1caccdb62928b18d70229R6

Reply:
Yes, but it imports its own custom hook from `./hooks` and not from `hooks`

Reply:
hrmmmm

Reply:
You are 💯 right, thanks for this nice catch!